### PR TITLE
updates to latest zipkin-reporter and armeria

### PR DESCRIPTION
**Pull request by codefromthecrypt**

### CODE REVIEW

File: armeria/pom.xml [hunk 1]
```diff
@@ -39,35 +39,35 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>2.0.62.Final</version>
+      <version>2.0.63.Final</version>
```
Comment (anuraaga) [inline armeria/pom.xml:42]: IIUC next netty release, and subsequent armeria release, we'll be able to remove this

**Review by anuraaga — APPROVED** [review]

File: armeria/pom.xml (hunk 1 above)
Comment (codefromthecrypt) [reply armeria/pom.xml:42]: looking forward to it!

Comment (codefromthecrypt) [reply armeria/pom.xml:42]: added the missing comment

**Comment by codefromthecrypt** [comment]
thanks for the look @anuraaga!

**Pull request merged**